libsolidity/ast/Types.h

```diff
@@ -81,7 +81,7 @@ class StorageOffsets
 public:
 	/// Resets the StorageOffsets objects and determines the position in storage for each
 	/// of the elements of @a _types.
-	void computeOffsets(TypePointers const& _types);
+	void computeOffsets(TypePointers const& _types, u256 _slotStart = 0);
```

Comment:
`_slotStart` should be documented and possibly renamed. Is it the base offset of the whole layout? Is it relative to parent?

The documentation for the class and other members should also clarify whether offsets are absolute or relative (and what they are relative to).